### DISCUSSION TITLE
Is there a way to build EdgeDB without the bundled Postgres?

### DISCUSSION THREAD

**Discussion by elliotwaite** [geldata/gel#6449] [Q&A] [answered]
If I am running EdgeDB in a Docker container, but using an external Postgres database, it seems like it would be beneficial to not also include the bundled Postgres in the Docker image. Is there a way to build EdgeDB without it? Or does EdgeDB still rely on the bundled Postgres code even when using an external Postgres database?

**Comment by fantix** [comment, marked as answer]
> Or does EdgeDB still rely on the bundled Postgres code even when using an external Postgres database?

I think this is unnecessarily true.

On the source code level, `get_default_runtime_params()` calls `buildmeta.get_pg_version()`, which runs `pg_config` command for the version. It's probably fine to hardcode Postgres version there, because we shouldn't rely on bundled Postgres version, it's nothing but a placeholder as in default runtime params. The real runtime params will fetch the version of the actual Postgres in use.

**Reply by fantix** [reply]
https://github.com/edgedb/edgedb/pull/3337

**Reply by elliotwaite** [reply]
Okay, got it. Would the EdgeDB team be open to removing this dependency so that a minimal Docker image could be created for use cases where an external Postgres is being used? I'd be open to helping out with the PR if needed, although it would probably take me some time to get familiar with the codebase.

**Comment by elprans** [comment]
I the image size the concern here?

**Reply by elliotwaite** [reply]
Yeah, reducing the image size would be the only benefit I can think of.
